test/image/mocks/bar_attrs_group_norm.json

```diff
@@ -0,0 +1,21 @@
+{
+    "data":[
+        {
+            "base":6,
+            "y":[1,2,3,4],
```

Comment:
@alexcjohnson I think making `y` mean `size` would be confusing, because I'd expect the values in `y` and the ticks in `yaxis` to be the same.